File: app/src/main/java/com/arman/queuetube/model/viewholders/PlaylistItemViewHolder.java
package com.arman.queuetube.model.viewholders;

import android.view.View;
import android.widget.ImageView;

import com.arman.queuetube.R;
import com.arman.queuetube.model.VideoData;
import com.arman.queuetube.model.adapters.VideoItemAdapter;

public class PlaylistItemViewHolder extends VideoViewHolder {

    public static final String TAG = "PlaylistItemViewHolder";

    public PlaylistItemViewHolder(View view) {
        super(view);
    }

    @Override
    public void bind(VideoData item, VideoItemAdapter.OnItemClickListener clickListener, VideoItemAdapter.OnItemDragListener dragListener) {
        super.bind(item, clickListener, dragListener);
    }

}

File: app/src/main/java/com/arman/queuetube/model/viewholders/PlaylistItemViewHolder.kt
package com.arman.queuetube.model.viewholders

import android.view.View
import com.arman.queuetube.model.VideoData
import com.arman.queuetube.model.adapters.BaseTouchAdapter

class PlaylistItemViewHolder(view: View) : VideoViewHolder(view) {

    override fun bind(item: VideoData, clickListener: BaseTouchAdapter.OnItemClickListener?, dragListener: BaseTouchAdapter.OnItemDragListener?) {
        super.bind(item, clickListener, dragListener)
    }

    companion object {

        val TAG = "PlaylistItemViewHolder"
    }

}
